Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/tsconfig.vstemplate

```diff
@@ -0,0 +1,15 @@
+<VSTemplate Version="3.0.0" Type="Item" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
+  <TemplateData>
+    <Name Package="{2ffe45c4-5c73-493c-b187-f2e955ff875e}" ID="1024"/>
+    <Description Package="{2ffe45c4-5c73-493c-b187-f2e955ff875e}" ID="1025"/>
+    <!-- pulled from the asp.net v5 package -->
```

Comment:
Yes, I checked with just Cordova and UWP installed.. and it works.. In TS repo, we always get tsconfig icon from this package.